Tests/TestCases.hpp

```diff
@@ -12,19 +12,41 @@
 #include <AMReX.H>
 
 // Test cases
+#include "BSSNMatterTest.hpp"
 #include "CCZ4GeometryUnitTest.hpp"
 #include "CCZ4RHSTest.hpp"
+#include "ChiRelaxationTest.hpp"
 #include "ConstraintsTest.hpp"
 #include "CoordinateTransformationsTest.hpp"
 #include "DerivativeUnitTests.hpp"
+#include "EMTensorTest.hpp"
+#include "MatterWeyl4Test.hpp"
 #include "PositiveChiAndAlphaUnitTest.hpp"
 #include "SphericalHarmonicTest.hpp"
 #include "Weyl4Test.hpp"
 
+TEST_CASE("BSSNMatter"
```

Comment:
For consistency with the other test names, let's use spaces.
```suggestion
TEST_CASE("BSSN Matter"
```

Reply:
I think the other test names should be changed to just the filenames/directory names as well as I keep forgetting what they are called.

Reply:
I think I would be happy to remove the spaces as it makes it easier to type out if you just want to run a single test case. However, I'll allow this so long as you open a PR to change the other test case names.

For reference, you can list the test-cases by doing
```
./Tests.<build config.ex -dt-l
```

Reply:
I have remove the white spaces from the other test cases :)